docs/deploying.md

```diff
@@ -1,40 +1,48 @@
 # Deploying
 
-When time comes to share your project, you have many options for deploying it for others to see. Framework is compatible with many static site hosts and automation environments. In this guide we’ll focus on deploying to Observable manually, then with GitHub Actions.
+You can host your built Framework project on any static site hosting service, or self-host it with any static site server. This guide covers deploying to [Observable](https://observablehq.com), which is the easiest way to host your Framework project as support is built-in. We’ll also cover setting up automated deploys with GitHub Actions.
 
-## Manually deploying to Observable
+<div class="tip">
 
-If you don’t already have a project to deploy, you can create one by following [getting-started](./getting-started). First, make sure that your project builds without error:
+If you don’t already have a project ready to deploy, create one by following our [Getting started guide](./getting-started).
+
+</div>
+
+## Manual deploys
+
+First, make sure that your project builds without error:
 
 ```sh
-$ npm run build
+npm run build
```

Comment:
We drop the `$` here because we want the copy button to work (copy from page, paste into terminal). Maybe in the future we could have a stylistic `$` that isn’t copied, but the shell-ness is also implied by the `sh` in the top-right corner.